source/guides/maintenance/increase-virtual-disk-size.rst

```diff
@@ -1,133 +1,140 @@
+For example, to find the size of the KVM image version number 31880,
 follow these steps:
 
 #. Go to the `releases`_ repository.
-#. Drill down into the `20450 > clear > config > image` directory.
-#. Open the :file:`hyperv-config.json` file.
-#. Locate the `PartitionLayout` key.
-
-   The example shows 512 MB for the EFI partition, 32 MB for the swap
-   partition, and 8 GB for the root partition.
+#. Drill down into the `31880 > clear > config > image` directory.
+#. Download and open the :file:`kvm.yaml` file.
+#. Locate the `targetMedia` section.
 
+   The example shows a total disk size of 8.54 GB, 512 MB for the EFI 
+   partition, 32 MB for the swap partition, and 8 GB for the root partition.
 
    .. code-block:: console
-
-      "PartitionLayout" : [ { "disk" : "hyperv.img",
-                              "partition" : 1,
-                              "size" : "512M",
-                              "type" : "EFI" },
-                              { "disk" : "hyperv.img",
-                              "partition" : 2,
-                              "size" : "32M",
-                              "type" : "swap" },
-                              { "disk" : "hyperv.img",
-                              "partition" : 3,
-                              "size" : "8G",
-                              "type" : "linux" } ],
+      :emphasize-lines: 3,9,13,18
+
+      targetMedia:
+      - name: ${bdevice}
+        size: "8.54G"
+        type: disk
+        children:
+        - name: ${bdevice}1
+          fstype: vfat
+          mountpoint: /boot
+          size: "512M"
+          type: part
+        - name: ${bdevice}2
+          fstype: swap
+          size: "32M"
+          type: part
+        - name: ${bdevice}3
+          fstype: ext4
+          mountpoint: /
+          size: "8G"
+          type: part
 
 Increase virtual disk size
 **************************
-Once you have determined the disk and partition to be increased, you are
-ready to perform the actual increase of the disk, partition, and filesystem.
 
-Power off VM and increase virtual disk size
-===========================================
+Follow these steps to increase the disk size, resize a partition, 
+and resize the filesystem: 
 
-To increase the virtual disk size for a prebuilt image, perform the steps
-below:
+Increase virtual disk size
+==========================
 
-#. Shut down your VM if it is running.
-#. Use the process defined by your hypervisor  or cloud provider to increase
+#. Shut down your VM.
+#. Use the process defined by your hypervisor or cloud provider to increase
    the virtual disk size of your |CL| VM.
-#. Power up the VM.
-
+#. Power up your VM.
 
-Resize the partition of the virtual disk
-========================================
+Resize the last partition of the virtual disk
+=============================================
 
-#. Log in to an account with root privileges.
-#. Open a terminal emulator.
+#. Log in.
+#. Open a terminal window.
```

Comment:
Currently, running the swupd bundle-add command below fails: 
Message:
`
Error: Failed to connect to update server: https://cdn.download.clearlinux.org/update/31890/Manifest.MoM.tar
Possible solutions for this problem are:
Check if your network connection is working
Fix the system clock
Run 'swupd info' to check if the urls are correct
Check if the server SSL certificate is trusted by your system ('clrtrust generate' may help)
Error: Failed to retrieve 31890 MoM manifest
Error: Cannot load official manifest MoM for version 31890
`

Reply:
I'll need to resolve this before I can go further. See email, please.

Reply:
BTW: I don't think the failure of the `swupd bundle-add ...` is related to this PR. 😦

Reply:
this error maybe a connectivity issue on your side.  works for me.